### ISSUE TITLE
Use sbt 1 to build Play itself

### ISSUE THREAD

**Issue by marcospereira** [playframework/playframework#8383]
### Play Version

Play 2.6+.

Maybe this won't be backported to Play 2.6, but there aren't huge differences between Play 2.6 and master branch regarding the build configuration, so we should be okay to backport.

### Expected Behavior

Play already cross build its plugin to sbt 0.13 and 1, but Play itself is not build using sbt 1. sbt tracks the progress of projects already using sbt 1 here:

https://github.com/sbt/sbt/wiki/library-sbt-1.x-migration

Not to be confused with [plugins migration](https://github.com/sbt/sbt/wiki/sbt-1.x-plugin-migration).

### Actual Behavior

Play is not built with sbt 1.

### Related

1. https://github.com/playframework/interplay/pull/43

**Comment by hepin1989** [comment]
Hope this happen in Play 2.7

**Comment by dwijnand** [comment]
I'm hopeful it happens during the RC-period feature freeze of 2.7, but that's just a personal hope.

**Comment by hepin1989** [comment]
seems like this will be apart of 2.8:)

**Comment by gaurang-sawhney** [comment]
Hi,
I would like to work on this issue. Should we upgrade to `1.2.8`?

**Comment by dwijnand** [comment]
Yes, we should use 1.2.8, and do the necessary migration away from private APIs.

**Comment by gaurang-sawhney** [comment]
@dwijnand I am trying to push the code but getting a forbidden(403) error
```
remote: Permission to playframework/playframework.git denied to gaurang-sawhney.                                                                                                      fatal: unable to access 'https://github.com/playframework/playframework.git/': The requested URL returned error: 403
```
Does anybody need to authorize me here?

**Comment by dwijnand** [comment]
@gaurang-sawhney you need to fork the repo and send a pull request: https://help.github.com/articles/about-pull-requests/

**Comment by dwijnand** [comment]
Fixed in #9001
